frontend/src/modules/app/ToolBar.tsx

```diff
@@ -113,13 +149,22 @@ export const ToolBar = ({
   const history = useHistory();
   const currentMode = useSelector(getParametersRequestMode);
   const backendInitialized = useSelector(getBackendInitialized);
-  const parameterRequestStandby = useSelector(getParametersRequestStandby, shallowEqual);
-  const alarmLimitsRequestStandby = useSelector(getAlarmLimitsRequestStandby, shallowEqual);
+  const parameterRequestDraft = useSelector(getParametersRequestDraft, shallowEqual);
   const ventilating = useSelector(getParametersIsVentilating);
+  const alarmLimitsRequestDraft = useSelector(getAlarmLimitsRequestDraft);
+  const alarmLimitsCurrent = useSelector(getAlarmLimitsCurrent);
+  const alarmLimitsUnsaved = useSelector(getAlarmLimitsUnsavedChanges);
```

Comment:
Similarly as above, this should be alarmLimitsRequestUnsaved (and maybe the selector should be named `getAlarmLimitsRequestUnsavedChanges` for clarity, though that makes it really long)

Reply:
using getAlarmLimitsRequestUnsaved instead